loafer/ext/aws/providers.py

```diff
@@ -3,10 +3,11 @@
 
 import botocore.exceptions
 
-from .bases import BaseSQSClient
 from loafer.exceptions import ProviderError
 from loafer.providers import AbstractProvider
 
+from .bases import BaseSQSClient
```

Comment:
isort complained about this. Maybe I'm missing some setting...

Reply:
Try this:
```
sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
no_lines_before=LOCALFOLDER
```

Reply:
@georgeyk I found it. The setting should be `known_local_folder`.

https://github.com/timothycrosley/isort#custom-sections-and-ordering